### DISCUSSION TITLE
BUSCA DE DATA DE ANIVERSÁRIO COM VARIÁVEL ANO

### DISCUSSION THREAD

**Discussion by jornalistaCarlosOsorio** [sqlitebrowser/sqlitebrowser#3682] [General]
SELECT 
UF,
CIDADE,
NOME,
SEXO,
DATA_NASCIMENTO,
DDDTELEFONE
FROM 01_listagem_feminina_nomes_unicos

Por favor,

Estou querendo filtrar as pessoas que fazem aniversário em um determinado dia. Podem me informar como faço para fazer uma busca, tendo como variante o ano 'YYYY'? 

Obrigado,

**Comment by justinclift** [comment]
Does this help?

https://github.com/sqlitebrowser/sqlitebrowser/wiki/Using-the-Filters

**Reply by jornalistaCarlosOsorio** [reply]
Eu não consegui. Preciso de uma variante nesta linha: WHERE DATA_ANIVERSÁRIO = 'VARIANTE-07-15'


SELECIONE
UF,
CIDADE,
NOME,
SEXO,
DATA_NASCIMENTO,
DDDTELEFONE
FROM 01_listagem_feminina_nomes_unicos
WHERE DATA_ANIVERSÁRIO = 'VARIANTE-07-15'

**Reply by justinclift** [reply]
Does it have to be a **filter** in the Browse Data tab, or would getting a result from using SQL in the "Execute SQL" tab be ok?
